guides/v2.1/cloud/docker/docker-database.md

```diff
@@ -0,0 +1,81 @@
+---
+group: cloud-guide
+title: Connect to the database
+functional_areas:
+  - Cloud
+  - Docker
+  - Configuration
+---
+
+There are two ways to connect to the database. Before you begin, you can find the database credentials in the `database` section of the `docker/config.php` file. The examples below use the following default credentials:
+
+> Filename: `config.php`
```

Comment:
Maybe will be better to provide path like docker/config.php because file with name config.php also exist in magento

Reply:
Please ignore it. I see you specified it in text

Reply:
I think you are right, anyway. I will update this. Good catch.